modules/upgrade/pages/migrate-4x-4x.adoc

```diff
@@ -74,6 +93,13 @@ spacewalk-service start
 
 If the process fails, check these issues first:
 
-* If Zypper  does not have the migration tool available, install the [package]``zypper-migration-plugin`` package.
+* If Zypper does not have the migration tool available, install the [package]``zypper-migration-plugin`` package.
```

Comment:
Same here, we are using `zypper-migration-plugin`, so renaming the procedures to `upgrade` seems to be confusing.